controller/src/main/java/io/pravega/controller/store/stream/StreamMetadataStore.java

```diff
@@ -1125,6 +1125,18 @@
     CompletableFuture<Map<UUID, ActiveTxnRecord>> getActiveTxns(final String scope, final String stream,
                                                                 final OperationContext context, final Executor executor);
 
+    /**
+     * Method to retrieve List of transaction in completed(COMMITTED/ABORTED) state
+     * from most recent batch.
+     *
+     * @param scope Scope of stream
+     * @param stream name of stream
```

Comment:
Capital letter after the variable name in all these params.

Reply:
Done as suggested.